src/views/Zap/__tests__/ZapStakeAction.unit.test.tsx

```diff
@@ -204,95 +119,24 @@ describe("<ZapStakeAction/> ", () => {
     fireEvent.click(await screen.getByText("Approve"));
     expect(await screen.findByText("Successfully approved")).toBeInTheDocument();
   });
-
-  it.skip("Should Display Error when unable to retrieve allowances", async () => {
```

Comment:
Why not test this?

Reply:
This test was specific to Zap slices. There is no longer a "An error has occurred when fetching token allowance." message that appears on the screen. I agree it's worthwhile to test, but the test behaves differently now and isn't specific to Zap (Function of TokenAllowanceGuard)